src/test/java/net/snowflake/ingest/streaming/internal/RowBufferTest.java

```diff
@@ -767,23 +407,22 @@ private void testFlushHelper(ArrowRowBuffer rowBuffer) {
     Assert.assertFalse(response.hasErrors());
     float bufferSize = rowBuffer.getSize();
 
-    ChannelData<VectorSchemaRoot> data = rowBuffer.flush();
+    ChannelData<?> data = rowBuffer.flush();
     Assert.assertEquals(2, data.getRowCount());
     Assert.assertEquals((Long) 1L, data.getRowSequencer());
-    Assert.assertEquals(7, data.getVectors().getFieldVectors().size());
+    // Assert.assertEquals(7, data.getVectors().getFieldVectors().size());
```

Comment:
?

Reply:
thanks, leftover, I moved it to `ArrowBufferTest#testFieldNumberAfterFlush()`